frontend/src/app/app.module.ts

```diff
@@ -6,11 +6,14 @@ import { AppComponent } from './app.component';
 import { CoreModule } from './core/core.module';
 import { SharedModule } from './shared/shared.module';
 import { ModalContentComponent } from './components/modal-content/modal-content.component';
-import { TestComponent } from './components/test/test.component';
+import { UserSettingsComponent } from './components/settings/user-settings/user-settings.component';
+import { SettingsComponent } from './components/settings/settings.component';
+import { ProjectSettingsComponent } from './components/settings/project-settings/project-settings.component';
+import { ReactiveFormsModule  } from '@angular/forms';
 
 @NgModule({
-  declarations: [AppComponent, ModalContentComponent, TestComponent],
-  imports: [BrowserModule, AppRoutingModule, CoreModule, SharedModule],
+  declarations: [AppComponent, ModalContentComponent, UserSettingsComponent, SettingsComponent, ProjectSettingsComponent],
+  imports: [BrowserModule, AppRoutingModule, CoreModule, SharedModule, ReactiveFormsModule],
```

Comment:
Add ReactiveFormsModule to the SharedModule in imports and exports in order do not import it every time